src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -37,10 +37,17 @@ def notifiable_link_text
       else
         "Removed as #{role} of a project"
       end
+    when 'Event::BuildFail'
+      project = @notification.event_payload['project']
+      package = @notification.event_payload['package']
+      repository = @notification.event_payload['repository']
+      arch = @notification.event_payload['arch']
+      "Package #{package} on #{project} project failed to build against #{repository} / #{arch}"
```

Comment:
This is hard to parse and distinguish notifications from one another, see this screenshot:

![hard-to-parse](https://user-images.githubusercontent.com/1102934/205064178-dae99860-b0ad-45b8-b916-70e9971f7a08.png)

Reply:
I think I'm going to improve over this in another PR because right now, I don't know what to add or remove to tell apart build fail event notifications from each other.

Reply:
I created [a card](https://trello.com/c/wpOJunpB/118-improve-how-to-tell-apart-multiple-build-failure-event-notifications) to improve this later